src/common/wallet-state.ts

```diff
@@ -359,13 +380,18 @@ function useWalletState(initialState?: Partial<WalletStateParams>): WalletData {
   //   return _.fromPairs(tokenBalances)
   // }
 
-  const loadBalance = async (transactions: Transaction[], currentBlock: number): Promise<void> => {
+  const loadBalance = async (transactions: Transaction[], _currentBlock: number): Promise<void> => {
```

Comment:
Why was it changed to not use `currentBlock`?
If somehow it's not needed anymore and using `latest` is fine, let's remove this parameter.